plugins/easings/elastic/README.md

```
@@ -0,0 +1,74 @@
+
+or
+
+```shell
+$ yarn add @tsparticles/plugin-easing-elastic
+```
+
+Then you need to import it in the app, like this:
+
+```javascript
+const { tsParticles } = require("@tsparticles/engine");
+const { loadEasingBackPlugin } = require("@tsparticles/plugin-easing-elastic");
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)